pkg/controller/rest/verifiable/operation_test.go

```diff
@@ -10,19 +10,26 @@ import (
 	"bytes"
 	"encoding/base64"
 	"encoding/json"
+	"errors"
 	"fmt"
+
 	"io"
 	"net/http"
 	"net/http/httptest"
 	"testing"
 
+	"github.com/hyperledger/aries-framework-go/pkg/doc/did"
+	"github.com/hyperledger/aries-framework-go/pkg/framework/aries/api/vdri"
```

Comment:
move this group of imports with the other framework imports

Reply:
done